lib/kimono_labs_client/differ.rb

```diff
@@ -0,0 +1,12 @@
+module KimonoLabsClient
+  # Provides an API for diffing POROs
+  class Differ
+    def self.diff(prev, curr)
+      diff = (prev.blank? || curr.blank? || prev.instance_variables.length != curr.instance_variables.length)
+
+      prev.instance_variables.each { |p| diff ||= (prev.send(p) != curr.send(p)) } unless diff
+
+      diff
+    end
```

Comment:
I'm not sure how I feel about using a class like this.  In the rest of the code, it will be:
```ruby
if Differ.diff(this, that)
```

Given the unitask nature of the entire class, I personally, would rather see something along the lines of:
```ruby
if Utils.diff(this, that)
```
Where `Utils` is a catch-all for the one-off class methods.

**OR**

Make `Differ` like this:
```ruby
class Differ
  def init(prev, curr)
    ...
  end
end
```
Where the object itself is the boolean value of the difference between the two things.  Then I wouldn't be so bothered by it.  You'd probably need to do some more meta programming around it to make sure the boolean-ness of it works correctly.